kruize-clowdapp.yaml

```
@@ -28,6 +28,26 @@ objects:
           limits:
             cpu: ${KRUIZE_CPU_LIMIT}
             memory: ${KRUIZE_MEMORY_LIMIT}
+        livenessProbe:
+          failureThreshold: 3
+          httpGet:
+            path: /health
+            port: ${AUTOTUNE_SERVER_PORT}
+            scheme: HTTP
+          initialDelaySeconds: 30
+          periodSeconds: 10
+          successThreshold: 1
+          timeoutSeconds: 3
+        readinessProbe:
+          httpGet:
+            path: /health
+            port: ${AUTOTUNE_SERVER_PORT}
+            scheme: HTTP
+          initialDelaySeconds: 3
```

```suggestion
          initialDelaySeconds: 10
```
Kruize service take a bit more time to start than ROS. In my EE testing readinessProbe was triggered while service was starting. It loads config, run migrations, loads some data into memory all this can take more than 3 sec. So better to have a initialDelay of 10 sec.

---

Ah okay, thanks